### ISSUE TITLE
New spew of E122 errors

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#201]
In GitLab by @rbtcollins on Nov 21, 2016, 16:10

```
pip install flake8
```

```
:!../../.tox/preflight/bin/flake8 --bug-report
{
  "dependencies": [
    {
      "dependency": "setuptools",
      "version": "28.8.0"
    }
  ],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.5.2",
    "system": "Linux"
  },
  "plugins": [
    {
      "plugin": "mccabe",
      "version": "0.5.2"
    },
    {
      "plugin": "pycodestyle",
      "version": "2.2.0"
    },
    {
      "plugin": "pyflakes",
      "version": "1.3.0"
    }
  ],
  "version": "3.2.1"
}
```

I had code that was linting happily and just starting spewing E122 errors all around the place. I note that 3.2.1 was just released. Coincidence? Maybe :p 

Example code:

```
    container_names = run_cmd(
        kubectl_argv(constants.KUBE_NAMESPACE) + [
        "get", "pod", pod,
        "--output", 'jsonpath="{.spec.containers[*].name}"'])
```

With output 

```
...
./.../infra_lib.py:281:9: E122 continuation line missing indentation or outdented
./.../infra_lib.py:282:9: E122 continuation line missing indentation or outdented
```

But that code looks entirely reasonable to me :)

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Nov 21, 2016, 16:47

So we just fixed an issue in 3.2.1 (#257) that was preventing some errors from being actually reported.

As for the code in question, I think pycodestyle wants:

```py
    container_names = run_cmd(
        kubectl_argv(constants.KUBE_NAMESPACE) + [
            "get", "pod", pod,
            "--output", 'jsonpath="{.spec.containers[*].name}"'])
```

**Comment by asottile** [comment]
In GitLab by @rbtcollins on Nov 21, 2016, 18:30

huh. Thanks! Not a bug I guess, though ugh.
